crates/binjs_io/src/entropy/read/content_decoders.rs

```diff
@@ -29,6 +29,11 @@ pub struct DictionaryStreamDecoder<'a, T> {
 
     /// The name of this dictionary. Used for debugging/error reporting.
     name: SharedString,
+
+    /// The latest index we have read from the prelude dictionary.
+    ///
+    /// Used to decode the special symbol `0`.
```

Comment:
can you explain what this value is used for?
here, or add reference to other method.

(IMO, it's more important than what the consumer is)